### PR TITLE
Fix lint warnings

### PR DESCRIPTION
**Pull request by isaul32**

### CODE REVIEW

**Comment by isaul32** [comment]
I think checks should fail if get any lint warning or error.

File: generators/server/templates/src/main/kotlin/package/security/SecurityUtils.kt.ejs [hunk 1]
```diff
@@ -144,25 +144,25 @@ fun isAuthenticated(): <% if (!reactive) { %>Boolean<% } else { %>Mono<Boolean><
...
 
-  if (authentication != null) {
-    val isUserPresent = getAuthorities(authentication)?.any { it == authority }
-    if(isUserPresent != null) {
-        return isUserPresent
+    if (authentication != null) {
+        val isUserPresent = getAuthorities(authentication)?.any { it == authority }
+        if(isUserPresent != null) {
+            return isUserPresent
+    }
```
Comment (sendilkumarn) [inline generators/server/templates/src/main/kotlin/package/security/SecurityUtils.kt.ejs:153]: nit: missing a tab here.

**Review by sendilkumarn — CHANGES_REQUESTED** [review]
Thanks @isaul32 for the PR.

**Review by sendilkumarn — APPROVED** [review]
Awesome! Thanks. Thanks for fixing other indentation issues too.

**Comment by sendilkumarn** [comment]
Congrats on your first contribution 🎉

**Pull request merged**